generate-stackbrew-library.sh

```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 declare -A release_channel=(
-	[stable]='16.0.6'
+	[stable]='17.0.1'
 	[production]='16.0.6'
```

Comment:
```suggestion
 	[production]='17.0.2'
```

Reply:
```suggestion
 	[production]='16.0.7'
```
the production channel is still on 16